echo-notifications/src/main/resources/templates/appReassigned/subject.vm

```diff
@@ -0,0 +1 @@
+The application $notification.additionalContext.application has been reassigned to you
```

Comment:
Not sure if it's worth putting a [SPINNAKER] prefix on the subject line so people can easily determine the originator?